### DISCUSSION TITLE
Airspy HF+ Support

### DISCUSSION THREAD

**Discussion by HFATL** [rtl-airband/RTLSDR-Airband#268] [General]
Can you please confirm that the Airspy HF+ is supported?

Thank you

**Comment by szpajder** [comment]
It is (via SoapySDR).

**Reply by HFATL** [reply]
Thank you!

**Comment by dlpwx** [comment]
@szpajder Is manual gain with Airspy HF+ now possible to support since SoapyAirspyHF enabled it?  [Soapy Changelog](https://github.com/pothosware/SoapyAirspyHF/blob/master/Changelog.txt)

**Reply by szpajder** [reply]
Yes, just use `--gain-elements`, as with any other receiver.
